public/message.js

```diff
@@ -0,0 +1,24 @@
+const { app, dialog } = require('electron')
+
+let isBoxOpen = false
+let userResponse = -1
+
+function showMessageBox(options, cb) {
+
+  if (isBoxOpen) {
+    return
+  }
+  else {
+    isBoxOpen = true
+  }
```

Comment:
No need for `else` here. Let's also rename the variable to `messageBoxOpen` instead of `isBoxOpen`.